src/compiler/commandLineParser.ts

```diff
@@ -488,13 +488,20 @@ namespace ts {
     }
 
     /* @internal */
-    export function parseListTypeOption(opt: CommandLineOptionOfListType, value: string, errors: Diagnostic[]): (string | number)[] {
-        const values = trimString((value || "")).split(",");
+    export function parseListTypeOption(opt: CommandLineOptionOfListType, value = "", errors: Diagnostic[]): (string | number)[] | undefined {
+        value = trimString(value);
+        if (startsWith(value, "-")) {
+            return undefined;
+        }
+        if (value === "" || value === ",") {

```

Comment:
I get the `--lib` part but the comma seems wierd though. not sure there is much value.